metadata_service/proxy/neo4j_proxy.py

```diff
@@ -592,14 +592,20 @@ def add_badge(self, *,
                   id: str,
                   badge_name: str,
                   category: str = '',
-                  resource_type: ResourceType = ResourceType.Table) -> None:
+                  resource_type: ResourceType,
+                  column_name: str = '') -> None:
 
         LOGGER.info('New badge {} for id {} with category {} '
                     'and resource type {}'.format(badge_name, id, category, resource_type.name))
 
         validation_query = \
             'MATCH (n:{resource_type} {{key: $key}}) return n'.format(resource_type=resource_type.name)
 
+        column_suffix = ''
+        if column_name:
+            column_suffix = '/' + column_name
```

Comment:
why not just reassign id here rather than add the + '' in case where column doesn't exist? Also can use interpolation rather than '/' + var so it's consistent with other code

Reply:
Ya I didn't know if it was best to change the input variable as others might want to use the variable for something else.